homeassistant/components/plex/media_player.py

```diff
@@ -378,7 +358,8 @@ def refresh(self, device, session):
                 self._device.proxyThroughServer()
             self._session = None
             self._machine_identifier = self._device.machineIdentifier
-            self._name = self._device.title or DEVICE_DEFAULT_NAME
+            self._name = '{} {}'.format('Plex', self._device.title or
+                                        DEVICE_DEFAULT_NAME)
```

Comment:
Since this is repeated, would the following make sense?
```python
NAME_FORMAT = 'Plex {}'
# ...
self._name = NAME_FORMAT.format(self._device.title or DEVICE_DEFAULT_NAME)
```

Reply:
Good call. It didn't feel quite right before.